lib/index.js

```diff
@@ -31,15 +31,15 @@ internals.implementation = function (server, options) {
       if (!token && request.auth.mode === 'optional') {
         return reply.continue({ credentials: {} });
       }
-      
+
       if (!token) {
-        return reply(Boom.unauthorized('Missing auth token'));
+        return reply(Boom.unauthorized(null, 'Token'));
```

Comment:
For our API we use JWT for user authenticated requests, but for services consuming the API we use an apiKey because we want to handle them differently.